src/pheval/resources/phenopackets/lirical/Abdul_Wahab-2016-GCDH-Patient_5.json

```diff
@@ -0,0 +1,208 @@
+{
+  "id": "PMID:27672653-Abdul_Wahab-2016-GCDH-Patient_5",
+  "subject": {
+    "id": "Patient 5",
+    "dateOfBirth": "1970-01-01T00:00:00Z",
+    "timeAtLastEncounter": {
+      "age": {
+        "iso8601duration": "P14Y"
+      }
+    },
+    "sex": "FEMALE",
+    "taxonomy": {
+      "id": "NCBITaxon:9606",
+      "label": "Homo sapiens"
+    }
```

Comment:
How big with the pheval package on pypi become if we merge all these in? Maybe better keep a corpus directory outside the `src` directory, like

```
testdata/phenopackets/lirical
```

Reply:
Just wondering what you are considering as binary data? Happy to move the Phenopacket directory to outside the src directory

Reply:
With binary file I meant in particular the huge tar.gz file you have added to this PR. binary data is not very amenable to git - you make a tiny change and the whole file changes. Anything like archive files (zip, gz) or other forms of packaged compressed data (jar files etc) should not be checked into git - at least not if the ever change (I guess, a one time commit with a guarantee it wont change in the next year could be justifiable)

Reply:
Ah yes, I understand now. The VCF file could perhaps be moved out of the src directory to the same testdata directory as the phenopackets. The VCF file itself won't ever be changed, it's just provided so that the variants in the Phenopacket can be spiked into a VCF to be run with Exomiser. Jules eventually will want 4 different types of VCF - genome and exome templates for both human genome builds so I guess it will be better to move it out of the src directory and here:

```
testdata/template_vcfs
```

Reply:
I guess thats fine, but my thinking was that the VCF file is provided by @souzadevinicius makefile rather than be checked into git? Maybe we discuss this tomorrow a moment.

Reply:
I'm going to restructure as suggested and try and get all the tests to pass with tox